src/time.ts

```
@@ -26,7 +26,7 @@ export class Time {
         date = date.toTimeString();
         break;
       case 'unix':
-        // TODO @Shinigami92 2022-01-10: add default case
+      default:
```

IMO this should throw an error if an unexpected value is given.

---

check for unexpected value added

---

Instead of adding a new if statement, you could just use the default case.

---

> Instead of adding a new if statement, you could just use the default case.

format = 'unix', ahh I got it